src/probnum/diffeq/steprule.py

```diff
@@ -75,16 +60,19 @@ class AdaptiveSteps(StepRule):
     """
 
     def __init__(
-        self, tol_per_step, localconvrate, limitchange=(0.1, 5.0), safetyscale=0.95
+        self,
+        tol_per_step,
+        localconvrate,
+        limitchange=(0.1, 5.0),
+        safetyscale=0.95,
+        **kwargs
     ):
         self.tol_per_step = float(tol_per_step)
         self.safetyscale = float(safetyscale)
         self.localconvrate = float(localconvrate + 1)
         self.limitchange = limitchange
 
     def suggest(self, laststep, errorest, **kwargs):
-        """
-        """
```

Comment:
While you're at it, would it be okay if you put a single sentence into this docstring instead of deleting the empty one? This is not criticism (!) but it would be cool to have something here that describes what ''suggest'' does :)

Reply:
`suggest` of the abstract base class is already documented and states the general purpose of the `suggest` function. Do you mean to describe more precisely what it does for `AdaptiveSteps`?

Reply:
wasn't aware that it is already documented in the superclass. In this case please ignore my comment.